### DISCUSSION TITLE
Tracing through multiple java services

### DISCUSSION THREAD

**Discussion by GokulShrikanth** [open-telemetry/opentelemetry-java-instrumentation#3260] [Q&A] [answered]
does starting the java applications with the openTelemetry jar automatically take care of the propagation and the continuation of the traces even when the applications are not on the same machine?

**Comment by trask** [comment, marked as answer]
hi @GokulShrikanth! yes, if you are using one of the instrumented libraries/frameworks to send and receive on both sides, and you are using the java agent on both applications, then distributed tracing header(s) should get propagated between them
